### PR TITLE
Ajout de traductions Fr (beta xcode 12)

### PR DESCRIPTION
**Pull request by iDreamsX**
Ajout de plusieurs éléments traduit en Français.

### CODE REVIEW

File: ACHNBrowserUI/ACHNBrowserUI/fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -489,7 +489,7 @@ You can cancel anytime with your iTunes account settings. Any unused portion of
 "On rotten turnips" = "Sur les navets pourris";
 "Shaking trees" = "En secouant les arbres";
 "On white flowers" = "Sur les fleurs blanches";
-"Flying near trash (boots, tires, cans)" = "Vole près des ordures (bottes, pneus, canettes)";
+"Flying near trash (boots, tires, cans) or rotten turnips" = "Vole près des ordures (bottes, pneus, canettes) ou prêt des navets pourris";
```
Comment (renaudjenny) [inline ACHNBrowserUI/ACHNBrowserUI/fr.lproj/Localizable.strings:492]: I'm not sure about this one as you changed the key, I think it won't work as expected. But the translation is good!
```suggestion
"Flying near trash (boots, tires, cans)" = "Vole près des ordures (bottes, pneus, canettes) ou prêt des navets pourris";
```

File: ACHNBrowserUI/ACHNBrowserUI/fr.lproj/Localizable.strings [hunk 2]
```diff
@@ -897,3 +897,37 @@ Merci d'être respecteux. Tout code signalé sera supprimé";
 "Very fast" = "Très rapide";
 "Fast" = "Rapide";
 "Stationary" = "Stationnaire";
+
+// Added 2020-06-07
+"Fishing tourney" = "Tournoi de pêche";
+"You turnips predictions will be displayed here one you fill in somes prices." = "Vos prévisions en vente de navets seront affichées ici, remplissez les prix pour l'afficher."
```
Comment (renaudjenny) [inline ACHNBrowserUI/ACHNBrowserUI/fr.lproj/Localizable.strings:903]: As I mentioned in the other PR, you could write
```suggestion
"You turnips predictions will be displayed here one you fill in somes prices." = "Vos prévisions de prix de vente de navets seront affichées ici, remplissez les prix pour l'afficher."
```

File: ACHNBrowserUI/ACHNBrowserUI/fr.lproj/Localizable.strings [hunk 3]
```diff
@@ -897,3 +897,37 @@ Merci d'être respecteux. Tout code signalé sera supprimé";
...
+"River (clifftop)" = "Rivière (falaise)"
+"River (mouth)" = "Rivière (Embouchure)"
+"On villagers" = "Sur les villageois"
+"On rivers/ponds" = "Sur les rivières/ponts"
+"Assessing fossils" = "Après expertisation des fossiles"
+"Cyrus" = "Serge"
+"Only avaible during Wedding Season" = "Disponible uniquement durant l'évent : Saisons des Mariages"
+"Bug-Off" = "Insectosafari"
+"K.K. concert" = "Concert de Kéké"
+"K.K. concert, Nook Shopping Catalog" = "Concert de Kéké,Nook Shopping (Sélection du jour)"
```
Comment (renaudjenny) [inline ACHNBrowserUI/ACHNBrowserUI/fr.lproj/Localizable.strings:924]:
```suggestion
"K.K. concert, Nook Shopping Catalog" = "Concert de Kéké, Nook Shopping (Sélection du jour)"
```

File: ACHNBrowserUI/ACHNBrowserUI/fr.lproj/Localizable.strings (hunk 1 above)
Comment (TheVaan) [reply ACHNBrowserUI/ACHNBrowserUI/fr.lproj/Localizable.strings:492]: The data changed and this fixes a broken translation

**Pull request merged**